### ISSUE TITLE
Title of Inquiry in nutshell

### ISSUE THREAD

**Issue by alexpelletier** [presidential-innovation-fellows/ceic-partnering#31]
Organization Name- ok
Tags beneath: show inquiry type (Connect to Expert, Partner with Lab) and add Expert/Tech Area, Lab/Add Lab
<img width="503" alt="screen shot 2016-02-16 at 4 50 44 pm" src="https://cloud.githubusercontent.com/assets/14278320/13092279/0af92cfa-d4cd-11e5-8e85-37c2ce74fe95.png">

**Comment by adamsfallen** [comment]
Yep will do. Going to update all of this as soon as @lukad03 implements how to approach form labels.

**Comment by adamsfallen** [comment]
Now support Inquiry Type (Connect to Expert, Partner with Lab and Other Questions). Can also support additional tags ala convention "tags: item, item2, item3" but would populate on the fly via other input fields w/js, which I defer to @lukad03 on how he prefers to handle.

**Comment by lukad03** [comment]
@adamsfallen When you get back lets sync up on this

**Comment by lukad03** [comment]
Adam and I chose to postpone this until after ARPA-E
